x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/indicator_match.ts

```diff
@@ -78,7 +79,7 @@ export const indicatorMatchExecutor = async ({
       eventsTelemetry,
       filters: ruleParams.filters ?? [],
       inputIndex,
-      itemsPerSearch: ruleParams.itemsPerSearch ?? 9000,
```

Comment:
I believe this should be `MAX_PER_PAGE`? If someone knows otherwise I can revert it back.